Source/HelixToolkit.Wpf.SharpDX.Tests/Elements3D/MeshGeometryModel3DTests.cs

```diff
@@ -9,20 +9,24 @@
 // --------------------------------------------------------------------------------------------------------------------
 
 using System;
+using System.Collections.Generic;
 using HelixToolkit.Wpf.SharpDX.Tests.Controls;
 using NUnit.Framework;
 using System.IO;
+using System.Threading;
+using SharpDX;
 
 namespace HelixToolkit.Wpf.SharpDX.Tests.Elements3D
 {
     [TestFixture]
+    [Apartment(ApartmentState.STA)]
     class MeshGeometryModel3DTests
     {
         /// <summary>
         /// Test for pull request #54.
         /// Fix IndexOutOfRange exception in CreateDefaultVertexArray.
         /// </summary>
-        [Test, STAThread]
```

Comment:
The tests didn't run in a STA Thread with this attribute, changed it to `[Apartment(ApartmentState.STA)]`